shardingsphere-distsql/shardingsphere-distsql-parser/src/main/java/org/apache/shardingsphere/distsql/parser/core/common/CommonDistSQLStatementVisitor.java

```diff
@@ -269,6 +270,11 @@ public ASTNode visitShowResources(final ShowResourcesContext ctx) {
         return new ShowResourcesStatement(null == ctx.schemaName() ? null : (SchemaSegment) visit(ctx.schemaName()));
     }
     
+    @Override
+    public ASTNode visitShowUnusedResources(final ShowUnusedResourcesContext ctx) {
+        return new ShowResourcesStatement(null == ctx.schemaName() ? null : (SchemaSegment) visit(ctx.schemaName()));
```

Comment:
The newly created statement should be used here instead of ShowResourcesStatement?

Reply:
> The newly created statement should be used here instead of ShowResourcesStatement?

@RaigorJiang Thanks for your review, I think the `Statement` content is the same between `SHOW RESOURCES` and `SHOW UNUSED RESOURCES`, so I want to share the class code.

Reply:
But the ResultSets is different, `RQLBackendHandlerFactory` needs to use the name of the statement to get the ResultSet SPI, the same statement will bring problem.

Reply:
> But the ResultSets is different, `RQLBackendHandlerFactory` needs to use the name of the statement to get the ResultSet SPI, the same statement will bring problem.

OK, I understand, I fix it.